src/Clip.cpp

```diff
@@ -713,6 +713,11 @@ string Clip::PropertiesJSON(int64_t requested_frame) {
 	root["has_audio"] = add_property_json("Enable Audio", has_audio.GetValue(requested_frame), "int", "", &has_audio, -1, 1.0, false, requested_frame);
 	root["has_video"] = add_property_json("Enable Video", has_video.GetValue(requested_frame), "int", "", &has_video, -1, 1.0, false, requested_frame);
 
+	root["crop_x"] = add_property_json("Crop X", crop_x.GetValue(requested_frame), "float", "", &crop_x, 0.0, 0.0, false, requested_frame);
+	root["crop_y"] = add_property_json("Crop Y", crop_y.GetValue(requested_frame), "float", "", &crop_y, 0.0, 0.0, false, requested_frame);
```

Comment:
Typo, I think. The 6th and 7th parameters to `add_property_json()` set the range for the parameter (min and max , respectively), so they can't both be the same value (`0.0`) or the parameter is unmodifiable.

Typically it'd be either `0.0`, `1.0` or `-1.0`, `1.0` depending how the value is used. @jonoomph suggested `0.0`, `1.0`, so I assume that's correct and the origin is the upper left corner. If the origin point were the center, `-1.0`, `1.0` would be more common. (Although it's worth discussing whether, for OpenShot, the origin _should_ be the center, as that's probably what people would be expecting?)